shpc/main/modules/__init__.py

```diff
@@ -219,17 +219,20 @@ def create_symlink(self, module_dir):
         # Create .version
         self.write_version_file(os.path.dirname(symlink_path))
 
-    def check_symlink(self, module_dir):
+    def check_symlink(self, module_dir, force):
```

Comment:
should probably have a default here! And +1 I like this change - it makes sense to have force here to mirror other commands!

```suggestion
    def check_symlink(self, module_dir, force=False):
```